skada/tests/test_reweight.py

```diff
@@ -61,6 +69,27 @@ def test_reweight_estimator(estimator, da_dataset):
     assert score > 0.9
 
 
+@pytest.mark.parametrize(
```

Comment:
either this test can be done with paramatrizing other estimator (or adding the estimator in another test) or it should have a specifics purpose and less generic name.

Reply:
It tests the TarS method on a regression problem. Sorry, I don't get the problem...